src/server.ts

```diff
@@ -1,10 +1,13 @@
 import * as Fastify from 'fastify'
+import { configureSwagger } from './plugins'
```

Comment:
I'd change the naming here so its a bit more declarative
```suggestion
import { configureSwaggerPlugin } from './plugins'
```

Reply:
Done.